tools/unix/build_omim.sh

```diff
@@ -90,6 +90,10 @@ fi
 # Find cmake
 source "$OMIM_PATH/tools/autobuild/detect_cmake.sh"
 
+if [ "${OPT_TARGET}" == "generator_tool" ]; then
+  CMAKE_CONFIG="${CMAKE_CONFIG:-} -DSKIP_QT_GUI=ON"
+fi
```

Comment:
As I mentioned before, we need to skip qt gui almost always, e.g. tools and all tests.
So this script should have -DSKIP_QT_GUI=ON by default, imo. And turned-off only in one case :)

Reply:
OK. Now `-DSKIP_QT_GUI` is `ON` by default.

Reply:
Ok, then remove this check?

Reply:
Done.